starlite/openapi/controller.py

```diff
@@ -95,12 +127,25 @@ def retrieve_schema_yaml(self, request: Request) -> "OpenAPI":
                 A [Request][starlite.connection.Request] instance.
 
         Returns:
-            A rendered YAML object..
+            resoponse: With a rendered YAML object
```

Comment:
`resoponse`.

Also, here too we need linkage, thus:

```suggestion
            A [Response][starlite.response.Response] instance with the YAML object rendered into a string.
```